src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -2,7 +2,8 @@ import { loc, createCallout } from 'okta';
 import { BaseForm } from '../../internals';
 import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
 
-const OV_NMC_FORCE_UPGRAGE_SERVER_KEY = 'idx.authenticator.app.method.push.force.upgrade.number_challenge';
+const OV_NMC_FORCE_UPGRAGE_SERVER_KEY = 'oie.authenticator.app.method.push.force.upgrade.number_challenge';
```

Comment:
Updated key to have `oie` prefix to be consistent with other keys  https://oktainc.atlassian.net/browse/OKTA-381041